packages/next/build/webpack-config.ts

```diff
@@ -78,9 +78,9 @@ function getOptimizedAliases(isServer: boolean): { [pkg: string]: string } {
     ),
 
     // Polyfill: Object.assign
-    __next_polyfill__object_assign: require.resolve('object-assign'),
     'object-assign$': stubObjectAssign,
     '@babel/runtime-corejs2/core-js/object/assign': stubObjectAssign,
+    '@babel/runtime-corejs2/core-js/promise': stubPromise,
```

Comment:
We should probably drop both this stub and the above object/assign stub with the new behavior.